app/controllers/application_controller.rb

```diff
@@ -11,9 +11,6 @@ class ApplicationController < ActionController::Base
 
   protect_from_forgery
 
-  #Authpds user functions
-  include Authpds::Controllers::AuthpdsController
-
   def connection_error
     render 'user_sessions/unexpected_error', :layout => false, :status => 500 and return
```

Comment:
@hab278 And change this location to whatever the new location of this template is i.e. `errors/unexpected_error`